humilis/layer.py

```diff
@@ -193,6 +194,8 @@ def compile(self):
             'Resources': self.section.get('resources', {}),
             'Outputs': self.section.get('outputs', {})
         }
+        if self.section.get('transform', {}).get('value', {}):
+            cf_template['Transform'] = self.section['transform']['value']
```

Comment:
Need to take into consideration the transform section from now on.
It can't be added just like another parameters (e.g `Outputs` or `Mappings`) because having a `Transform` section with `{}` as value in a CF template make the deployment fails. This was the reason why classic layers (that is to say no-SAM layers) were failing.